library/Preprocessor.php

```diff
@@ -44,18 +44,18 @@ class Webgrind_Preprocessor
      */
     static function parse($inFile, $outFile)
     {
+        // If possible, use the binary preprocessor
+        if (self::binaryParse($inFile, $outFile)) {
+            return;
+        }
```

Comment:
What is the reason for moving this above the r/w checks?

Reply:
On Windows, the w+b handle prevented the binary from working.  This seemed like the simplest change to fix.

The binary will still fail if it cannot read or write (which is why the exit code check was added), so the same exceptions will get thrown if the files cannot be accessed.

-[Unknown]